tests/integration/server.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
+
+# Copyright 2016 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import time
+
+from google.cloud import logging as gcloud_logging
+from google.cloud import monitoring as gcloud_monitoring
+from google.cloud.monitoring import MetricKind, ValueType
+from google.cloud.exceptions import Forbidden as ForbiddenException
+from google.cloud.exceptions import NotFound as NotFoundException
+
+from flask import Flask, request, jsonify
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def hello_world():
+    return 'Hello World!'
+
+
+@app.route('/logging', methods=['POST'])
+def _logging():
+    request_data = request.get_json()
+    if request_data is None:
```

Comment:
flask should automatically raise a `BadRequest` exception if you just `get_json()` and the request isn't valid.